.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,80 @@
+name: Continuous Integration
```

Comment:
```suggestion
name: CI
```

Can we use something shorter? The checks can't even fit the word "Successful" 😧 

![image](https://user-images.githubusercontent.com/229881/73109221-8f81f580-3ed0-11ea-9391-67557ddf9beb.png)

Reply:
I guess, I just usually try to avoid abbreviations :D